webapp/components/features/FollowList/FollowList.story.js

```diff
@@ -0,0 +1,111 @@
+import { storiesOf } from '@storybook/vue'
+import { withA11y } from '@storybook/addon-a11y'
+import { action } from '@storybook/addon-actions'
+
+import helpers from '~/storybook/helpers'
+import FollowList from './FollowList.vue'
+
+import fuzzyFilterUser from './FollowList.story.json'
+
+helpers.init()
+
+const user = {
+  name: 'Jenny Rostock',
+  id: 'u3',
+  followedByCount: 12,
+  followedBy: helpers.fakeUser(7),
+  followingCount: 28,
+  following: helpers.fakeUser(7),
+}
+
+const lessThanSevenUser = {
+  ...user,
+  followedByCount: 3,
+  followedBy: user.followedBy.slice(0, 3),
+  followingCount: 3,
+  following: user.following.slice(0, 3),
+}
+
+const allConnectionsUser = {
+  ...user,
+  followedBy: [...user.followedBy, ...helpers.fakeUser(5)],
+  following: [...user.following, ...helpers.fakeUser(21)],
+}
+
+const noConnectionsUser = {
+  ...user,
+  followedBy: [],
+  followedByCount: 0,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T14:29:15Z" title="Tuesday, April 7th 2020, 4:29:15 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,123 @@
+import { storiesOf } from '@storybook/vue'
+import { withA11y } from '@storybook/addon-a11y'
+import { action } from '@storybook/addon-actions'
+
+import helpers from '~/storybook/helpers'
+import FollowList from './FollowList.vue'
+
+import fuzzyFilterUser from './FollowList.story.json'
+
+helpers.init()
+
+const user = {
+  name: 'Jenny Rostock',
+  id: 'u3',
+  followedByCount: 12,
+  followedBy: helpers.fakeUser(7),
+  followingCount: 28,
+  following: helpers.fakeUser(7),
+}
+
+const allConnectionsUser = {
+  ...user,
+  followedBy: [...user.followedBy, ...helpers.fakeUser(5)],
+  following: [...user.following, ...helpers.fakeUser(21)],
+}
+
+const noConnectionsUser = {
+  ...user,
+  followedBy: [],
+  followedByCount: 0,
+  following: [],
+  followingCount: 0,
+}
+
+const wrapTemplates = (templates) => `
+<div style="display: flex; flex-wrap: wrap;">
+${templates.map((template) => `<div style="margin: 8px;">${template}</div>`).join('')}
+</div>
```


I don't think this is necessary. I would be happy with just having one `FollowList` component in Storybook instead of two, side-by-side. Because it's the (single) component that's in focus here – how it is used in the app shouldn't make a difference. Or what was your thinking behind this?

If you want to show the different texts for the different `types` I'd suggest to just add a story for that.

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:18:24Z" title="Thursday, April 9th 2020, 11:18:24 pm +02:00">Apr 9, 2020</time>_

---


7ff9d617d